packages/ai/src/microsoft/teams/ai/agent.py

```diff
@@ -0,0 +1,27 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+"""
+
+from typing import Any, Awaitable, Callable
+
+from .ai_model import AIModel
+from .chat_prompt import ChatPrompt, ChatSendResult
+from .function import Function
+from .memory import Memory
+from .message import Message
+
+
+class Agent(ChatPrompt):
```

Comment:
could we add a desc here that notes this is a stateful vers of ChatPrompt?

Reply:
Yes!